src/Compilation.zig

```diff
@@ -880,24 +880,16 @@ pub fn create(gpa: *Allocator, options: InitOptions) !*Compilation {
             break :blk false;
         };
 
-        const darwin_can_use_system_linker_and_sdk =
+        const darwin_can_use_system_sdk =
             // comptime conditions
             ((build_options.have_llvm and comptime std.Target.current.isDarwin()) and
             // runtime conditions
             (use_lld and std.builtin.os.tag == .macos and options.target.isDarwin()));
```

Comment:
Since this requires `use_lld` to be true and since `use_lld` will now, if I understand correctlly, always be false when targeting darwin, can we get rid of this logic to maybe use the system sdk entirely?

Reply:
Not yet, since we still use `use_lld` to select between incremental MachO linker and `Zld` so this logic will have to say until I converge the two implementations which are currently separate.